src/rgw/rgw_file.cc

```diff
@@ -121,10 +121,7 @@ namespace rgw {
 	auto ux_key = req.get_attr(RGW_ATTR_UNIX_KEY1);
 	auto ux_attrs = req.get_attr(RGW_ATTR_UNIX1);
 	if (ux_key && ux_attrs) {
-	  DecodeAttrsResult dar = rgw_fh->decode_attrs(ux_key, ux_attrs);
-	  if (get<0>(dar) || get<1>(dar)) {
-	    update_fh(rgw_fh);
-          }
+	  [[maybe_unused]] DecodeAttrsResult dar = rgw_fh->decode_attrs(ux_key, ux_attrs);
```

Comment:
i've grown fond of std::ignore to silence compiler warnings like this,
```c++
std::ignore = rgw_fh->decode_attrs(ux_key, ux_attrs);
```

Reply:
I feel used to the pragma, I guess